app/assets/javascripts/ajax_mixpanel_events.js

```diff
@@ -0,0 +1,35 @@
+var ajaxMixpanelEvents = (function() {
+  var init = function() {
+    $("[data-track-click]").on("click", function (e, options = {}) {
+      if (options.trackedEvent === true) {
+        return;
+      }
+
+      e.preventDefault();
+      e.stopPropagation();
+
+      var clickedElement = $(e.target);
+      var eventData = {
+        event_name: "clicked_link",
+        controller_action: window.mixpanelData.controller_action,
+        full_path: window.mixpanelData.full_path,
+        data: {
+          link_text: clickedElement.text().trim(),
+        }
+      };
+
+      $.ajax({
+        type: "POST",
+        url: "/ajax_mixpanel_events",
+        timeout: 200,
```

Comment:
I would put this number higher. The risk of moving on too early is that we mistake timeouts as a lack of interest in signing in.

Reply:
This timed out for me locally